src/dataflow/src/source/util.rs

```diff
@@ -85,12 +86,14 @@ where
         // `capabilities` should be a two-element vector.
         let secondary_capability = capabilities.pop().unwrap();
         let data_capability = capabilities.pop().unwrap();
-        let durability_capability = CapabilitySet::from_elem(data_capability.clone());
+        let durability_capability1 = CapabilitySet::from_elem(data_capability.clone());
+        let durability_capability2 = CapabilitySet::from_elem(data_capability.clone());
```

Comment:
i think this is meant to be `secondary_capability.clone()`?

Reply:
Thank you. That solved probably a lot of debugging on my part.